scripts/o3de/o3de/repo_properties.py

```diff
@@ -354,6 +372,8 @@ def add_parser_args(parser):
                                                   description='path arguments to use with the --add-gems or --replace-gems option')
     modify_gems_group.add_argument('--release-archive-path','-rap',  type=pathlib.Path, required=False,
                             help='Create a release archive at the specified local path and update the download_source_uri and sha256 fields.')
+    modify_gems_group.add_argument('--force', '-f', action='store_true', default=False,
+                                                help='Copies over instantiated template directory even if it exist.')
```

Comment:
"Copies over instantiated template directory even if it exist." is that accurate? This is used so that the release_archive (zip) will still be created even if a zip already exists (overwrite the existing zip)